src/modules/form/data/referralPosting.json

```diff
@@ -0,0 +1,27 @@
+{
+  "item": [
+    {
+      "type": "CHOICE",
+      "linkId": "status",
+      "required": false,
+      "text": "Select Referral Status",
```

Comment:
```suggestion
      "text": "Referral Status",
```

We don't usually put "select" in the status. It will be added to the placeholder.

Reply:
Make sense, I picked it up from the screenshot of the mock in https://www.pivotaltracker.com/story/show/185276711